modules/svg/data.js

```diff
@@ -475,7 +475,7 @@ export function svgData(projection, context, dispatch) {
             d3_text(url)
                 .then(function(data) {
                     drawData.setFile(extension, data);
-                    var isTaskBoundsUrl = extension === '.gpx' && url.indexOf('project') > 0 && url.indexOf('task') > 0;
```

Comment:
@gaoxm @zlavergne @jeffdefacto this is one thing we definitely need to discuss- is this change okay to make? Seems like it would totally break stuff, but I see the value in allowing somebody to just send .gpx data without a project/task associated with it.

Reply:
In that case we will be unsure about whether their gpx URL is for the purpose of defining the boundary of a mapping task; e.g. some users may load GPS traces through gpx URL and use them to guide mapping roads. I'm leaning towards keeping the checks.